finalProject/highLevelCC.md

```diff
@@ -0,0 +1,81 @@
+## Ceasar Cipher
+
+My initial thoughts for solving this problem were:
+- I would need an array of the alphabet.
+- I would need a function called encode.
+- I have no idea what else to do and I should google this problem.
+
+I found a ton of different solutions, but a lot of them were using libraries, which I'm not at all familiar with using. The most basic solution I could find was solved something like this. _You can find this in finalProject/CC.js_
+
+- Create a function called caeserShift that takes a string and an amount.
+
+  ```js
+  var caesarShift = function (str, amount) {
+
+  }
+```
+
+
+ - You need to ensure that your numbers are not negative by adding 26 if the amount argument is a negative number. This allows us to have shifts that go right or left in the alphabet.
+
+  ```
```

Comment:
This solution uses a concept known as `recursion` which is a much higher level skill that you will learn more about in Mod 4! I'm glad that you used your resources to try to find a solution that worked, but if you cannot fully understand or explain what is going on, that should be a flag that it is code you shouldn't use in your own work.